hardhat.config.ts

```diff
@@ -102,7 +111,20 @@ const config: HardhatUserConfig = {
         currency: "USD",
     },
     etherscan: {
-        apiKey: ETHERSCAN_API_KEY,
+        apiKey: {
+            goerli: ETHERSCAN_API_KEY ?? "",
+            polygonMumbai: ETHERSCAN_API_KEY ?? "",
+        },
+        customChains: [
```

Comment:
please write in comments why do we need custom chain

Reply:
we don't need that